src/aspire/denoising/__init__.py

```diff
@@ -21,26 +20,12 @@ def __init__(self, src):
 
     def denoise(self):
         """
-        Denoise 2D images
+        Precompute for Denoiser and DenoisedImageSource for 2D images
         """
         raise NotImplementedError('subclasses must implement this')
 
-    def to_src(self):
+    def images(self):
```

Comment:
Signature (arguments) should match the one in the subclass.

Reply:
Done.